util/envDynawo.sh

```diff
@@ -1294,6 +1587,211 @@ deploy_dynawo_autocompletion() {
   $DYNAWO_HOME/util/autocompletion/deploy_autocompletion.sh $*
 }
 
+build_doc() {
+  if [ ! -d "$DYNAWO_HOME/documentation" ]; then
+    error_exit "$DYNAWO_HOME/documentation does not exist."
+  fi
+  cd $DYNAWO_HOME/documentation
+  bash dynawo_documentation.sh
+}
+
+clean_doc() {
+  if [ ! -d "$DYNAWO_HOME/documentation" ]; then
+    error_exit "$DYNAWO_HOME/documentation does not exist."
+  fi
+  cd $DYNAWO_HOME/documentation
+  bash clean.sh
+}
+
+open_doc() {
+  if [ -x "$(command -v xdg-open)" ]; then
+    xdg-open $DYNAWO_HOME/documentation/dynawoDocumentation/DynawoDocumentation.pdf
+  elif [ ! -z "$DYNAWO_PDFVIEWER" ]; then
+    if [ -x "$(command -v $DYNAWO_PDFVIEWER)" ]; then
+      $DYNAWO_PDFVIEWER $DYNAWO_HOME/documentation/dynawoDocumentation/DynawoDocumentation.pdf
+    else
+      error_exit "$DYNAWO_PDFVIEWER seems not to be executable."
+    fi
+  else
+    error_exit "Cannot determine how to open pdf document from command line. Use DYNAWO_PDFVIEWER environement variable."
+  fi
+}
+
+build_nrt_doc() {
+  if [ ! -d "$DYNAWO_HOME/nrt/documentation" ]; then
+    error_exit "$DYNAWO_HOME/nrt/documentation does not exist."
+  fi
+  cd $DYNAWO_HOME/nrt/documentation
+  bash merge_tnr_descriptions.sh
+}
+
+clean_nrt_doc() {
+  for folder in $(find $DYNAWO_HOME/nrt/data -type d -name "description"); do
+    if [ -d "$folder" ]; then
+      (cd $folder; rm -f *.toc *.aux *.bbl *.blg *.log *.out *.pdf *.gz *.mtc* *.maf *.lof)
+    fi
+  done
+  (cd $DYNAWO_HOME/nrt/documentation; rm -f *.toc *.aux *.bbl *.blg *.log *.out *.pdf *.gz *.mtc* *.maf *.lof nrt_doc.tex)
+}
+
+open_nrt_doc() {
+  if [ -x "$(command -v xdg-open)" ]; then
```

Comment:
same as above. Also we could factorize this code

Reply:
ok